build_tools/docker/manage_images.py

```diff
@@ -50,6 +50,7 @@
     "riscv": ["base"],
     "gradle-android": ["base"],
     "frontends": ["android"],
+    "shark": [],
```

Comment:
Any reason we don't want to build from base image?

Reply:
SHARK has its own setup scripts to install all dependencies so it only needs a minimal set (git, Python) to run. Using base might mask any dependencies that SHARK should be installing.